streams/src/main/java/org/apache/kafka/streams/state/QueryableStoreType.java

```diff
@@ -48,4 +48,20 @@
      */
     T create(final StateStoreProvider storeProvider,
              final String storeName);
+
+    /**
+     * Create an instance of {@code T} (usually a facade) that developers can use
+     * to query the underlying {@link StateStore}s.
+     *
+     * @param storeProvider     provides access to all the underlying StateStore instances
+     * @param storeName         The name of the Store
+     * @param queryStaleState   Whether to allow querying recovering and standby stores.
```

Comment:
`recovering` -> `restoring`?  (recovering talks about task state and not store state?)